### PR TITLE
Add v0.16.3 release from spack

### PR DESCRIPTION
**Pull request by fangohr**
Offer the new Spack version (0.16.3) as additional workflows.

The failures for OSX are pretty random. We have not changed anything relative to the current main branch. I would thus suggest to merge this PR, and to accept that OSX workflows are (for now) shown as failures.

Long term perspective: if we can't fix the OSX compile, we could move it into a dedicated branch, so that we get a 'green tick' for commits for which the Linux build passes.

Additional observation: the OSX build for 0.16.3 is running for longer than 0.16.2, and stops with OOMMF reporting a problem from an internal check. In some way, this may be progress as we haven't made it to that point before (in a container).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,15 +12,22 @@ Compile OOMMF on (Debian) Linux with spack:
...
-- Compile OOMMF on OSX with Spack:
+- Compile OOMMF on OSX with Spack, see `osx` branch at see https://github.com/fangohr/oommf-in-spack/tree/osx
 
   [![osx-spack-v0.16.2-plus-epsilon](https://github.com/fangohr/oommf-in-spack/actions/workflows/osx-spack-v0.16.2.yml/badge.svg?branch=osx)](https://github.com/fangohr/oommf-in-spack/actions/workflows/osx-spack-v0.16.2.yml)
+  
+    [![osx-spack-v0.16.3](https://github.com/fangohr/oommf-in-spack/actions/workflows/osx-spack-v0.16.3.yml/badge.svg?branch=osx)](https://github.com/fangohr/oommf-in-spack/actions/workflows/osx-spack-v0.16.3.yml)
+
 
 
 # Required commands
```
Comment (lang-m) [inline README.md:33]: Colon/no colon before commands seems to be somewhat randomly in this section.

File: README.md [hunk 2]
```diff
@@ -12,15 +12,22 @@ Compile OOMMF on (Debian) Linux with spack:
 
 [![debian-spack-v0.16.2](https://github.com/fangohr/oommf-in-spack/actions/workflows/debian-spack-v0.16.2.yml/badge.svg)](https://github.com/fangohr/oommf-in-spack/actions/workflows/debian-spack-v0.16.2.yml)
 
-For information: (not part of this `main` branch)
+[![debian-spack-v0.16.3](https://github.com/fangohr/oommf-in-spack/actions/workflows/debian-spack-v0.16.3.yml/badge.svg)](https://github.com/fangohr/oommf-in-spack/actions/workflows/debian-spack-v0.16.3.yml)
+
+
+
+Compile OOMMF on OSX with Spack's `develop` branch (see branch https://github.com/fangohr/oommf-in-spack/tree/osx for these workflows):
```
Comment (lang-m) [inline README.md:19]: I don't quite understand the arrangement of workflow badges. Additionally, some of the given information seems to be repetitive/inconsistent.

File: README.md [hunk 3]
```diff
@@ -12,15 +12,22 @@ Compile OOMMF on (Debian) Linux with spack:
...
 
 - Spack `develop` branch, see https://github.com/fangohr/oommf-in-spack/tree/spack-develop
   [![debian-spack-develop](https://github.com/fangohr/oommf-in-spack/actions/workflows/debian-spack-develop.yml/badge.svg?branch=spack-develop)](https://github.com/fangohr/oommf-in-spack/actions/workflows/debian-spack-develop.yml)
```
Comment (lang-m) [inline README.md:22]: This is on Debian despite the "headline" above saying OSX.

File: README.md [hunk 4]
```diff
@@ -12,15 +12,22 @@ Compile OOMMF on (Debian) Linux with spack:
...
 [![debian-spack-v0.16.2](https://github.com/fangohr/oommf-in-spack/actions/workflows/debian-spack-v0.16.2.yml/badge.svg)](https://github.com/fangohr/oommf-in-spack/actions/workflows/debian-spack-v0.16.2.yml)
 
-For information: (not part of this `main` branch)
+[![debian-spack-v0.16.3](https://github.com/fangohr/oommf-in-spack/actions/workflows/debian-spack-v0.16.3.yml/badge.svg)](https://github.com/fangohr/oommf-in-spack/actions/workflows/debian-spack-v0.16.3.yml)
+
+
+
+Compile OOMMF on OSX with Spack's `develop` branch (see branch https://github.com/fangohr/oommf-in-spack/tree/osx for these workflows):
 
 - Spack `develop` branch, see https://github.com/fangohr/oommf-in-spack/tree/spack-develop
```
Comment (lang-m) [inline README.md:21]: Using bullet points here/no bullet points above makes the whole section somewhat inconsistent.

File: README.md (hunk 2 above)
Comment (lang-m) [inline README.md:19]: Could we maybe say that there are some additional branches for testing "more specific use cases" (`develop`, `osx`) that are (for simplicity or information purposes ?) listed below?

File: README.md (hunk 4 above)
Comment (lang-m) [reply README.md:21]: The use of bullet points ultimately depends on what exactly is show in the headline above.

File: README.md (hunk 2 above)
Comment (fangohr) [reply README.md:19]: Yes, sorry, I am working on it. We can talk this pm.

**Comment by lang-m** [comment]
One of my comments doesn't show up in the review above:
The use of colon/no colon in the section "Required commands" seems to be somewhat random.

**Pull request merged**